### PR TITLE
Nc news4

### PR DESCRIPTION
**Pull request by Ryanlee2690**

### CODE REVIEW

File: controllers/news.controller.js [hunk 1]
```diff
@@ -42,4 +42,12 @@ exports.getTopics = (req, res, next) => {
     .catch((err) => {
         next(err);
     })
-  };
\ No newline at end of file
+  };
+
+  exports.commentCounter = (req, res, next) => {
```
Comment (Rosem10) [inline controllers/news.controller.js:47]: I don't think this is needed, as all of our comment counting work is now being done in our sql statement in the model.

File: app.js [hunk 2]
```diff
@@ -15,6 +15,7 @@ app.patch('/api/articles/:article_id', patchVotesById)
 app.get('/api/users', getUsernames)
 
 
+
```
Comment (Rosem10) [inline app.js]: Remove some of this whitespace please!

File: models/news.model.js [hunk 3]
```diff
@@ -8,7 +8,10 @@ exports.selectAllTopics = () => {
 };
 
 exports.lookupArticleById = (id) => {
-    return db.query('SELECT * FROM articles WHERE article_id = $1', [id])
+    return db.query(`SELECT articles.article_id, articles.title, articles.topic, articles.author, articles.body, articles.created_at, articles.votes,
```
Comment (Rosem10) [inline models/news.model.js]: Perhaps consider breaking this out across more lines to make it more readable.

**Review by Rosem10 — CHANGES_REQUESTED** [review]
Bit of tidying up to be done, but otherwise really nice!

File: controllers/news.controller.js (hunk 1 above)
Comment (Ryanlee2690) [reply controllers/news.controller.js:47]: Whoops, remnants of some testing I forgot to remove

**Pull request merged**